.eslintrc.js

```diff
@@ -0,0 +1,97 @@
+module.exports = {
```

Comment:
As an alternative you can add ` "eslintConfig": { ... }` to your `package.json`. TIL.